packages/shared/src/components/icons/Key/KeyReferral/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react';
+import Icon, { IconProps } from '../../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+export const KeyReferralIcon = (props: IconProps): ReactElement => (
+  <Icon {...props} IconPrimary={FilledIcon} IconSecondary={OutlinedIcon} />
```

Comment:
Is the outlined correct? looks like a big blur?

Reply:
Let me check the asset.

Reply:
After checking, yes, that is expected. That's the disabled state.